include/eve/module/core/regular/countl_one.hpp

```diff
@@ -7,10 +7,24 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/detail/overload.hpp>
+#include <eve/arch.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
+#include <eve/module/core/regular/countl_zero.hpp>
+#include <bit>
 
 namespace eve
 {
+  template<typename Options>
+  struct countl_one_t : elementwise_callable<countl_one_t, Options>
+  {
+    template<eve::value T>
```

Comment:
All functions must be on `eve::unsigned_value`

Reply:
done